Bolts/iOS/BFAppLinkReturnToRefererView.h

```diff
@@ -23,25 +25,6 @@ typedef NS_ENUM(NSUInteger, BFIncludeStatusBarInSize) {
 };
 
 /*!
- Protocol that a class can implement in order to be notified when the user has navigated back
- to the referer of an App Link.
- */
-@protocol BFAppLinkReturnToRefererViewDelegate <NSObject>
```

Comment:
This doesn't seem necessary, as it's only relevant for usage together with BFAppLinkReturnToRefererView.